src/Elmish.WPF/Elmish.WPF.fsproj

```diff
@@ -1,9 +1,25 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I was wondering about this line while working on the multi-targeting.  I suppose Paket uses this to help parse the file.

Reply:
I guess Paket adds that just due to the XML parsing implementation it uses – same reason it removes spaces in the project files, I suppose. Those Paket edits always annoy me.